### ISSUE TITLE
Simplified Today weather overview 

### ISSUE THREAD

**Issue by neomodern** [martykan/forecastie#158]
Something like this (the highlighted part):
![overview](http://i.imgur.com/Wvv02eM.png)

It can be swiped to view the whole length of the day, though I understand the best accuracy we can get via free APIs is 3 hours rather than hourly. 

This could be as a setting to switch on/off in the settings, either replacing part of the current view or adding to it. 

Right now there's a lot of data on the UI which myself (and I'm sure many users) don't care for - I would rather get an immediate overview of temperature and likelihood of rain more than anything. So there should be some way to disable elements of the interface also.

**Comment by artem-p** [comment]
Agree.

**Comment by c64bob** [comment]
Absolutely

**Comment by jobbb** [comment]
Agree too

**Comment by ramack** [comment]
I also agree, that the view could be improved. In principle I like the vertical scrolling through the day, but the weekday and the date is far less important than the time in "today" and "tomorrow" views.

In the "later" view, the granularity is too fine in my eyes. 1 row per day would be perfect, but including minimum temperature.

I cannot really imagine wind of strength "3.3m/s", I would prefer to have a rough description like "none", "little", "medium", "storm", "strong storm" (I don't know the English words that are commonly used).

Maybe it could be configurable what to display and whether the details (scientifica values as currently) a rough description, both or nothing is shown in all the different views?

And last but not least, I think colored icons for sun, moon and clouds would be very good.

**Comment by robinpaulson** [comment]
@ramack Perhaps the Beaufort scale would be best?

The idea is fine in principle, but not simple to do. Any help appreciated.

**Comment by OffifialMITX** [comment]
Maybe you could also give us an option to show a combined graph instead of the regular view? I'll make a picture if you want to see what I'm thinking of.

**Comment by robinpaulson** [comment]
@OffifialMITX Yes, please do. Mock-ups are always a great start for visual changes.

**Comment by robinpaulson** [comment]
@OffifialMITX The combined graph part of this now has its own issue #297
